packages/strapi-admin/admin/src/containers/Webhooks/ListView/tests/reducer.test.js

```diff
@@ -0,0 +1,146 @@
+import { fromJS } from 'immutable';
+import reducer from '../reducer';
+
+describe('Admin | containers | Webhooks | ListView | reducer', () => {
+  const initialState = fromJS({
+    webhooks: [
```

Comment:
I think it would be great if this is not initialised. `webhooks: []` so you run your tests like your component behave